src/Dali/MetadataReader.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Minormous\Metabolize\Dali;
```

Comment:
Namespace should not include Metabolize, just Minormous\Dali

Reply:
But this is in the Metabolize project. Everything in this project is prefixed with `Minormous\Metabolize`.

Reply:
ugh disregard. Was thinking of the other PR